### DISCUSSION TITLE
挂载百度网盘，可下载文件到PC，但上传文件提示：找不到该项目

### DISCUSSION THREAD

**Discussion by chine369** [AlistGo/alist#2581] [Q&A]
**Alist Version / Alist 版本**
Built At: 2022-11-11 11:47:24 +0000
Commit ID: efe0e6a
Version: v3.4.0
WebVersion: 3.4.0

**Driver used / 使用的存储驱动**
百度网盘/raidrive 2020.11.30及最新版、rclone

**Describe the bug / 问题描述**

挂载百度网盘，可下载文件到PC，但上传文件提示：找不到该项目。可以在本地盘符中新建文件夹。更换了raidrive和rclone软件都可100%复现问题。

挂载阿里云盘，无上述问题。

和Issues中的这个问题很相似。【139 和彩云 个人云网盘挂载到本地盘符，无法复制上传文件】https://github.com/alist-org/alist/issues/1764

![alist上传问题](https://user-images.githubusercontent.com/18133981/205547736-408279f9-b230-49bd-98a2-bb9b06537b76.PNG)

![alist上传问题2](https://user-images.githubusercontent.com/18133981/205548565-67b2ae5a-aece-44f5-989f-7ef6185bce40.PNG)

**Comment by kokojacket** [comment]
请问您解决了吗 我也遇到这个问题了

**Comment by sundys** [comment]
我的百度,天翼,移动全部这个提示,微软,123都正常

**Reply by kokojacket** [reply]
官方手册里找到问题原因了，Raidrive 是因为上传文件会预先创建一个空文件 而百度网盘不允许创建空文件。估计天翼移动都是这个问题。

**Reply by kokojacket** [reply]
无解

**Reply by sundys** [reply]
我换了rclone挂载,问题全部解决,一切都正常了.果然是raidrive的坑导致的.

**Reply by kokojacket** [reply]
我是纯小白，reclone不会用阿🤣

**Comment by youtehub** [comment]
raidrive 新版本有些问题，还要付费
